cmd/virt-v2v-monitor/virt-v2v-monitor.go

```diff
@@ -57,6 +120,35 @@ func updateProgress(progressCounter *prometheus.CounterVec, disk, progress uint6
 	return
 }
 
+func xmlHandler(w http.ResponseWriter, r *http.Request) {
+	var xmlConf Domain
+	err := xml.Unmarshal([]byte(xmlData.String()), &xmlConf)
```

Comment:
I don't know if it's worth it, but since we only need one field, it may make sense to avoid loading the entire XML into the memory and instead iterate over it using `xml.Decoder` and get it directly